Missing include for GEngine when packaging from source

### What happened?

```bash
[3/8] Compile [x64] CesiumCreditSystem.cpp
C:\MyProject\\Plugins\CesiumForUnreal\Source\CesiumRuntime\Private\CesiumCreditSystem.cpp(147): error C2065: 'GEngine': undeclared identifier
C:\MyProject\\Plugins\CesiumForUnreal\Source\CesiumRuntime\Private\CesiumCreditSystem.cpp(147): error C2059: syntax error: ')'
```

### Environment

Cesium for Unreal version: v2.16.0
Unreal Engine Version: 5.5.4
Operating System: Win64


### Reproduction steps

_No response_

### Supporting evidence

_No response_

--- a/CHANGES.md
+++ b/CHANGES.md
@@ -1,5 +1,11 @@
 # Change Log {#changes}
 
+### ???
+
+##### Fixes :wrench:
+
+- Add a missing include for `GEngine` when packaging from source, introduced in *v2.16.0*.
+
 ### v2.16.0 - 2025-05-01
 
 ##### Additions :tada:

--- a/Source/CesiumRuntime/Private/CesiumCreditSystem.cpp
+++ b/Source/CesiumRuntime/Private/CesiumCreditSystem.cpp
@@ -5,6 +5,7 @@
 #include "CesiumCreditSystemBPLoader.h"
 #include "CesiumRuntime.h"
 #include "CesiumUtility/CreditSystem.h"
+#include "Engine/Engine.h"
 #include "Engine/World.h"
 #include "EngineUtils.h"
 #include "ScreenCreditsWidget.h"

